tools/querytee/proxy_metrics.go

```diff
@@ -45,11 +46,17 @@ func NewProxyMetrics(registerer prometheus.Registerer) *ProxyMetrics {
 			Help:      "Total number of responses compared per route name by result.",
 		}, []string{"route", "result"}),
 		relativeDuration: promauto.With(registerer).NewHistogramVec(prometheus.HistogramOpts{
-			Namespace: queryTeeMetricsNamespace,
-			Name:      "backend_request_relative_duration_seconds",
-			Help:      "Time (in seconds) of preferred backend less secondary backend.",
-			Buckets:   []float64{-100, -50, -25, -10, -5, -4, -3, -2, -1.5, -1, -.75, -.5, -.25, -.1, -.05, -.025, -.01, -.005, .005, .01, .025, .05, .1, .25, .5, 0.75, 1, 1.5, 2, 3, 4, 5, 10, 25, 50, 100},
-		}, []string{"method", "route"}),
+			Namespace:                   queryTeeMetricsNamespace,
+			Name:                        "backend_request_relative_duration_seconds",
+			Help:                        "Time (in seconds) of preferred backend less secondary backend.",
+			NativeHistogramBucketFactor: 2,
+		}, []string{"route"}),
+		proportionalDuration: promauto.With(registerer).NewHistogramVec(prometheus.HistogramOpts{
+			Namespace:                   queryTeeMetricsNamespace,
+			Name:                        "backend_request_relative_duration_proportional",
+			Help:                        "Proportional time (%) of preferred backend vs secondary backend.",
```

Comment:
[nit] Perhaps this would be clearer?

```suggestion
			Help:                        "Response time of preferred backend, as a proportion of secondary backend response time.",
```